source/Halibut.Tests/Support/BackwardsCompatibility/LatestClientAndPreviousServiceVersionBuilder.cs

```diff
@@ -178,6 +178,7 @@ async Task<IClientAndService> IClientAndServiceBuilder.Build(CancellationToken c
         
         public async Task<ClientAndService> Build(CancellationToken cancellationToken)
         {
+            var logger = new SerilogLoggerBuilder().Build().ForContext<LatestClientAndPreviousServiceVersionBuilder>();
```

Comment:
For now, this logger is only needed in the `else if (serviceConnectionType == ServiceConnectionType.PollingOverWebSocket)` section, right?